frontend/app/src/i18n/cn.json

```diff
@@ -647,10 +647,10 @@
         "joined_group": "加入群组",
         "nothingDone": "你还有很多工作要做！",
         "nothingLeftToDo": "放松！你没什么可做的了",
-        "owned_group_with_1000_diamond_members": "拥有一个拥有 1000 名经过验证的成员的群组/社区",
-        "owned_group_with_100_diamond_members": "拥有一个拥有 100 名经过验证的成员的群组/社区",
-        "owned_group_with_10_diamond_members": "拥有一个拥有 10 名经过验证的成员的群组/社区",
         "owned_group_with_1_diamond_member": "拥有一个拥有 1 名经过验证的成员的群组/社区",
+        "owned_group_with_10_diamond_members": "拥有一个拥有 10 名经过验证的成员的群组/社区",
```

Comment:
Balls. Looks like the "merge-latest-translations" function orders these differently to sync_lang